docs/reference-guides/block-api/block-bindings.md

```diff
@@ -0,0 +1,296 @@
+				'label'              => __( 'Visualization Date', 'custom-bindings' ),
+				'get_value_callback' => function( array $source_args, $block_instance ) {
+					$post_id = $block_instance->context['postId'];
+					$visualization_date = get_post_meta( $post_id, 'wp_movies_visualization_date', true );
+					if ( ! $visualization_date ) {
+						$visualization_date = date("m/d/Y");
+					}
+					return $visualization_date;
+				},
+				'uses_context'       => array( 'postId' ),
+			)
+		);
+	}
+);
+```
+
+This example needs a `post_meta` registered:
+
+```php
+add_action(
+	'init',
+	function() {
+		register_meta(
+			'post',
+			'wp_movies_visualization_date',
+			array(
+				'show_in_rest' => true,
+				'single'       => true,
+				'type'         => 'string',
+				'label'	       => __( 'Movie visualization date' ),
+			)
+		);
+	}
+);
+```
+
+#### Block bindings source value filter
+
+The value returned by `get_value_callback` can be modified with the `block_bindings_source_value` filter.
+The filter has the following parameters:
+
+- `value`: The value to be filtered.
+- `name`: The name of the source.
+- `source_args`: Array containing source arguments.
+- `block_instance`: The block instance object.
+- `attribute_name`: The name of the attribute.
+
+#### Server registration Core examples
+
+There are a few examples in Core that can be used as a reference.
+
+- Post Meta. [Source code](https://github.com/WordPress/wordpress-develop/blob/68d2421d758498095449006a055ad8ce5b7a8a40/src/wp-includes/block-bindings/post-meta.php#L59)
+- Pattern overrides. [Source code](https://github.com/WordPress/wordpress-develop/blob/68d2421d758498095449006a055ad8ce5b7a8a40/src/wp-includes/block-bindings/pattern-overrides.php#L36)
+- Twenty Twentyfive theme. [Source code](https://github.com/WordPress/wordpress-develop/blob/68d2421d758498095449006a055ad8ce5b7a8a40/src/wp-content/themes/twentytwentyfive/functions.php#L130)
+
+
+### Editor registration
+
+Editor registration on the client allows defining what the bound block will do when the value is retrieved or when the value is edited.
+
+The function to register a custom source is `registerBlockBindingsSource( args )`.
+
+- `args` is an `object` with the following structure:
+    - `name` is a `string` with the unique and machine-readable name.
+    - `label` is a `string` with the human readable name of the custom source. (optional)
+    - `usesContext` is an `array` with the block context that the custom source may need. (optional)
+    - `getValues` is a `function` that retrieves the values from the source. (optional)
+    - `setValues` is a `function` that allows updating the values connected to the source. (optional)
+    - `canUserEditValue` is a `function` to determine if the user can edit the value. The user won't be able to edit by default. (optional)
+    - `getFieldsList` is a private `function`. It cannot be used yet by third-party developers. It creates a list for the block bindings UI with post meta.
```

Comment:
Since `getFieldsList` can in fact be used by third-party developers, should we say something like the following instead?

    - `getFieldsList` is an experimental `function` that should not be used yet by third-party developers in production, as it is subject to change and may be removed in upcoming releases. It creates a list for the block bindings UI with post meta.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/66251/commits/2c19ec0a78b180f72d8c55e502cf361095c06354